src/Microsoft.NetCore.Analyzers/Core/Security/DoNotSetSwitch.cs

```diff
@@ -28,15 +28,15 @@ public sealed class DoNotSetSwitch : DiagnosticAnalyzer
             nameof(SystemSecurityCryptographyResources.DoNotDisableSchUseStrongCrypto),
             nameof(SystemSecurityCryptographyResources.DoNotDisableSchUseStrongCryptoMessage),
             DiagnosticHelpers.EnabledByDefaultIfNotBuildingVSIX,
-            helpLinkUri: null,
+            helpLinkUri: "https://docs.microsoft.com/visualstudio/code-quality/ca5361",
```

Comment:
Can you also run CIBuild locally so the documentation files are updated? Otherwise, I can do it once your change goes in...

Reply:
Also tagging @JoeRobich for some auto-generated PR help here. Context: We have some analyzer documentation files that are checked into the repo, and these get auto-updated when cibuild.cmd is executed locally. However, it is not reasonable to expect every dev or PR to run this locally prior to submitting a PR. Our current model is that I periodically execute this locally on a new branch and submit a PR, but it would be good if we can setup automated PRs (similar to merge PRs across branches) where the PR just does the following:

1. Fetch the latest `master` branch sources and git checkout a new local branch.
2. Execute `eng\common\cibuild` locally
2. If there are any diffs in the source tree, submit a PR to roslyn-analyzers repo with the change. Otherwise, bail out without submitting a PR.

Having the above automated PR generation execute at the same frequency as automated branch merge PRs would suffice. @JoeRobich let us know if this is something that can be setup - I can set it up if you provide instructions.

Reply:
An example PR is here: https://github.com/dotnet/roslyn-analyzers/pull/2676. I just executed `eng\common\cibuild` on latest master sources, pushed a commit on a new branch and submitted a PR.

Reply:
@mavasani That seems reasonable and doable. I created a issue in roslyn-tools to track it. https://github.com/dotnet/roslyn-tools/issues/529